src/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx

```diff
@@ -1,12 +1,79 @@
 import React from "react"
 import { Flex, color, Box, ResponsiveImage, Sans } from "@artsy/palette"
-import styled from "styled-components"
 import { NavBarHeight } from "Components/NavBar"
+import { Media } from "Utils/Responsive"
 
 export const ViewingRoomHeader: React.FC = props => {
   return (
-    <HeaderContainer>
-      <ImageArea>
+    <>
+      <Media greaterThanOrEqual="sm">
+        <ViewingRoomHeaderLarge />
+      </Media>
+      <Media lessThan="sm">
+        <ViewingRoomHeaderSmall />
+      </Media>
```

Comment:
These layouts are different enough to warrant two different components (I think -- might revise this later)